Sources/Swift/Core/Helper/Log/SentryLevel.swift

```diff
@@ -1,36 +1,14 @@
 import Foundation
 
-@objc
-public enum SentryLevel: UInt {
-    static let levelNames = ["none", "debug", "info", "warning", "error", "fatal"]
-    
-    @objc(kSentryLevelNone)
-    case none = 0
-    
-    // Goes from Debug to Fatal so possible to: (level > Info) { .. }
-    @objc(kSentryLevelDebug)
-    case debug = 1
-    
-    @objc(kSentryLevelInfo)
-    case info = 2
-    
-    @objc(kSentryLevelWarning)
-    case warning = 3
-    
-    @objc(kSentryLevelError)
-    case error = 4
-    
-    @objc(kSentryLevelFatal)
-    case fatal = 5
-}
+let levelNames = ["none", "debug", "info", "warning", "error", "fatal"]
```

Comment:
`m`: Shouldn't that be private?
```suggestion
private let levelNames = ["none", "debug", "info", "warning", "error", "fatal"]
```